### PR TITLE
tmp fix for exercise 9

### PR DESCRIPTION
**Pull request by adekbadek**
temporary fix for issue #92 (and #81), a legit fix would have to re-implement [comparestdout.js](https://github.com/workshopper/workshopper-exercise/blob/master/comparestdout.js), but just to compare first two lines

### CODE REVIEW

File: exercises/throw_an_error/exercise.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 var exercise      = require('workshopper-exercise')()
   , filecheck     = require('workshopper-exercise/filecheck')
   , execute       = require('workshopper-exercise/execute')
-  , comparestdout = require('workshopper-exercise/comparestdout')
+  , split         = require('split')
+  , chalk         = require('chalk')
+  , tuple         = require('tuple-stream')
+  , through2      = require('through2')
```
Comment (TimothyGu) [inline exercises/throw_an_error/exercise.js]: These dependencies are not added to packages.json.

File: exercises/throw_an_error/exercise.js [hunk 2]
```diff
@@ -11,7 +17,21 @@ exercise = filecheck(exercise)
 exercise = execute(exercise)
 
 // compare stdout of solution and submission
-exercise = comparestdout(exercise)
+exercise.addProcessor(function (mode, callback) {
+  let outputStream = through2.obj(function (chunk) {
+    const result = chunk[0] === chunk[1]
```
Comment (TimothyGu) [inline exercises/throw_an_error/exercise.js]: Don't use ES2015 `let` and `const`. `var` would do.

Comment (adekbadek) [inline exercises/throw_an_error/exercise.js]: I changed that, but what's the reason?

**Comment by TimothyGu** [comment]
Thank you!

File: exercises/throw_an_error/exercise.js (hunk 2 above)
Comment (TimothyGu) [inline exercises/throw_an_error/exercise.js]: We still support old Node.js I think

**Pull request merged**